spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java

```diff
@@ -179,13 +185,29 @@ ArmeriaServerConfigurator actuatorServerConfigurator(
             ManagementServerProperties serverProperties,
             BeanFactory beanFactory,
             ArmeriaSettings armeriaSettings) {
-        final EndpointMapping endpointMapping = new EndpointMapping(properties.getBasePath());
+        // default base-path is empty, so we can just concat it
+        final EndpointMapping endpointMapping =
+                new EndpointMapping(serverProperties.getBasePath() + properties.getBasePath());
```

Comment:
Should we make this only for management port only and exclude from internal service port?
e.g.
```
armeria:
  ports:
  - port: 8080
    protocols:
    - http
  internal-services:
    port: 8081
    protocols: https
  ssl:
    enabled: true

management:
  server:
    port: 8082
    address: 127.0.0.1
    base-path: /bar
    ssl:
      enabled: true
```

For port 8082, we will become `/bar/actuator/`. And how about 8081? should we use `/bar/actuator/` or `/actuator`?

As the issue at https://github.com/codecentric/spring-boot-admin/issues/1723#issuecomment-846142051, looks like it's better to use `/actuator` for internal-services' port?

Reply:
> looks like it's better to use /actuator for internal-services' port?

+1 for this.
Currently, we do not have any relationship between `management.server.base-path` and the path of internal services. We might consider moving `internal-services` under `management.server` property later.